include/exadg/structure/time_integration/driver_steady_problems.cpp

```diff
@@ -124,8 +124,8 @@ DriverSteady<dim, Number>::do_solve()
     // calculate right-hand side vector
```

Comment:
Note that in the nonlinear case above, we have to pass `param.update_preconditioner` to even allow updates, as the Newton solver checks with an && every *n* iterations or at the end, if the preconditioner should be updated. We could do a follow-up PR to get rid of all the head-to-head calls of the preconditioner update...
For my target applications (nonlinear time-dependent problems + FSI), the effect is probably not sooo big.